### ISSUE TITLE
[Vision] ImagePredictor.evaluate does not support pd.DataFrame input

### ISSUE THREAD

**Issue by Innixma** [awslabs/autogluon#1125]
`ImagePredictor.evaluate(dataset)` crashes if `dataset` is a `pd.DataFrame`.

```
AttributeError: ‘DataFrame’ object has no attribute 'transform_first'
```

**Comment by talhaanwarch** [comment]
Until this issue resolve, saving csv and loading it back is workaround
```
dataset.to_csv('dataset.csv',index=False)
dataset=ImagePredictor.Dataset.from_csv('dataset.csv')
ImagePredictor.evaluate(dataset)
```

**Comment by Innixma** [comment]
Resolved via #1164
